programs/C/length_string.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+
+//Created a function to calculate length of string.
+int LengthOfString(char string[50])
+{
+	
+	scanf("%[^\n]" , string);					//Input of string with blank spaces
+	int length = strlen(string);
```

Comment:
`strlen` is defined in `string.h`. Calculate length of string using loops instead of built-in functions